src/components/admin/hotels/manage-hotels.js

```diff
@@ -1,10 +1,72 @@
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
-
-export default function ManageHotels() {
-  return (
-    <Typography paragraph>
-      The 'Manage Hotels' Page comming soon...
-    </Typography>
-  );
-}
\ No newline at end of file
+import {DataGrid} from '@material-ui/data-grid'
+import NoItem from '../../common/no-item';
+import SearchPlugin from '../../common/search-plugin';
+import {seedData, columns} from './manage-hotels-constants';
+
+class ManageHotels extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            hotelsRows: [],
+            filteredList: [],
+            searchTerm: ''
+        };
+    }
+
+    componentDidMount() {
+        // getting data from API
+        const hotelsRows = seedData;
+
+        this.setState({
+            hotelsRows: hotelsRows,
+            filteredList: hotelsRows
+        });
+    }
+
+    filterList = (searchTherm) => {

```

Comment:
Please use class methods, instead of passing arrow functions, when you write a class-based component